.pending/improvements/gaiarest/align-swagger-file-t

```diff
@@ -0,0 +1 @@
+align swagger file to run contract tests
```

Comment:
I don't understand this pending log entry...not very informative imho. Perhaps we either remove it or state what's actually of tangible use to the consumer?

Reply:
Can be removed